packages/block-editor/src/hooks/typography.js

```diff
@@ -231,7 +231,7 @@ export function TypographyPanel( props ) {
 	);
 }
 
-const hasTypographySupport = ( blockName ) => {
+export const hasTypographySupport = ( blockName ) => {
```

Comment:
Why this change? You haven't used anywhere else this export.

Reply:
Yeah, I changed my mind :)